carla_world_node:
  ros__parameters:
    dt: 0.05
    # map_name: Town06
    map_name: big_straight_highway.xodr
    use_custom_map: true
    host: carla
    port: 2000
    